dracut/30coreos-installer/coreos-installer.service

```diff
@@ -1,15 +1,9 @@
 [Unit]
 Description=CoreOS Installer
-Requires=network-online.target
-After=network-online.target
-After=dracut-initqueue.service
-Before=dracut-pre-mount.service
-ConditionKernelCommandLine=coreos.inst=yes
-OnFailure=emergency.target
-OnFailureIsolate=yes
+After=coreos-installer.target
 
 [Service]
-Type=oneshot
+Type=simple
```

Comment:
Any reason for this change?

Reply:
Essentially, this service is the main process in this installer "runlevel". Think of it like `gdm.service` or something. One this service is started, systemd's job is done as far as booting things up.